### ISSUE TITLE
(question) Cuda 11.3 / 11.4 / 11.5

### ISSUE THREAD

**Issue by kiatchun** [lightvector/KataGo#566]
Are there currently any plans to support cuda version 11.3 / 11.4 / 11.5 for the Cuda and TensorRT versions?

**Comment by michito744** [comment]
On Windows 10, katago-v1.10.0-trt8.2-cuda11.2-windows-x64 is working properly in the following environment.

TensorRT 8.2.0.6
cuDNN 8.2.4.15
CUDA 11.4.2

Since TensorRT8.2 requires CUDA11.4+cuDNN8.2, I haven't tried CUDA11.5 in the first place.

**Comment by kiatchun** [comment]
I think the name katago-v1.10.0-trt8.2-cuda11.2-windows-x64 should be changed to katago-v1.10.0-trt8.2-cuda11.4(and whatever other versions is supported)-windows-x64

**Comment by lightvector** [comment]
> Since TensorRT8.2 requires CUDA11.4+cuDNN8.2,

No, TensorRT 8.2 supports all of CUDA 11.1 through 11.4. See this documentation:
https://docs.nvidia.com/deeplearning/tensorrt/archives/tensorrt-820-ea/support-matrix/index.html#platform-matrix

To avoid the risk of problems, I would strictly follow nvidia's documentation here, and only use combinations they list as supported. They have not added CUDA 11.5 to the table yet. Also in general, I would not rush to upgrade, as I mentioned here: https://github.com/lightvector/KataGo/issues/568. If you want stability, it may be preferable to wait for new CUDA versions to be tested and used a bit more instead of upgrading to them immediately when they are released - often CUDA versions get an "update 1" or an "update 2" following their initial release that fixes some more bugs.

> I think the name katago-v1.10.0-trt8.2-cuda11.2-windows-x64 should be changed to katago-v1.10.0-trt8.2-cuda11.4(and whatever other versions is supported)-windows-x64

I named it according to what CUDA version I used when I compiled it. If it works with other CUDA versions too, great. But I disagree, I think it is not a good idea to name it differently than the version that I actually used and tested when compiling.
